cloud/envs/aws.py

```diff
@@ -0,0 +1,24 @@
+from cloud.envs import env
+from cloud.envs import registry
+from cloud.envs import utils
+
+
+@registry.register("aws")
+class AzureInstance(env.Instance):
+
+  def __init__(self, config, **kwargs):
+    super().__init__(**kwargs)
+    self.access_key = config["access_key"]
+    self.secret_key = config["secret_key"]
+
+  @property
+  def driver(self):
+    if getattr(self, '_driver', None) is None:
+      self._driver = get_driver(Provider.EC2)(self.access_key, self.secret_key)
```

Comment:
Provider.AZURE

with this init scheme
```
driver = cls(subscription_id='subscription-id',
             key_file='/path/to/azure_cert.pem')
```

Reply:
lol, got confused by the class name. this should work for AWS

Reply:
lol oooopps